packages/contracts/README.md

```diff
@@ -1,10 +1,50 @@
-# grants-round/contracts
+# Allo Contracts
 
-This project is built using [hardhat](https://hardhat.org).
-All the contracts required to run / manage a round can be found here.
-If you come across any vulnerabilties, do create an issue / raise a PR to help improve the contracts. ^_^
+Allo Protocol is a set of smart contracts that enable the democratic allocation
+and distribution of funds through the power of [Quadratic
+Funding](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3243656). It is the
```

Comment:
This isn't entirely true 
- centralised Gitcoin used pairwise QF while we use a light weight linear QF
- Allo doesn't restrict itself to just QF as it's build to allow folks to add their own voting strategies like QV / Maci. Is it worth rephrasing that to state it supports add-ons but we chose to lead with QF